packages/fast-components-react-base/src/managed-classes.ts

```diff
@@ -0,0 +1,13 @@
+ export type ClassNames<T> = {
+    /**

```

Comment:
Should this comment be above our export statement?

Reply:
sure, moved